docs/packages.toml

```
@@ -151,6 +151,9 @@
     user = "MadNLP"
     rev = "389561354a774441cd5e6b3aa5fffee102ed222e"
     has_html = true
+[Manopt]
+    user = "JuliaManifolds"
+    rev = "v0.4.42"
```

```suggestion
[Manopt]
    user = "JuliaManifolds"
    rev = "v0.4.42"
    filename = "Readme.me"
```

If I read the code correctly, this might fix the docs error?